packages/@divvi/mobile/src/tokens/utils.ts

```diff
@@ -194,3 +194,28 @@ export function isHistoricalPriceUpdated(token: TokenBalance) {
 export function isFeeCurrency(token: TokenBalance | undefined): token is TokenBalance {
   return token?.isNative || !!token?.isFeeCurrency || !!token?.feeCurrencyAdapterAddress
 }
+
+export function getSerializableTokenBalance(tokenInfo: TokenBalance): SerializedTokenBalance {
+  const { balance, priceUsd, lastKnownPriceUsd, historicalPricesUsd, ...baseToken } = tokenInfo
+
+  return {
+    ...baseToken,
+    historicalPricesUsd,
+    balance: balance.toString(),
+    priceUsd: priceUsd?.toString() ?? null,
+    lastKnownPriceUsd: lastKnownPriceUsd?.toString() ?? null,
+  }
+}
+
+export function getDeserializedTokenBalance(
```

Comment:
supernit: just to follow what we did for prepared transactions.
```suggestion
export function getTokenBalance(
```